### PR TITLE
Swift 5

### PR DESCRIPTION
**Pull request by stephencelis**
This is the Swift 5-ready release!

### CODE REVIEW

**Comment by stephencelis** [comment]
Dang. Linux-only compiler bug: https://bugs.swift.org/browse/SR-10168

I think we can "fix" by un-nesting types that are in generic contexts, like: https://github.com/pointfreeco/swift-html/blob/master/Sources/Html/Attributes.swift#L404-L409

Not great...but otherwise we're kinda blocked.

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2018 Brandon Williams
+Copyright (c) 2019 Point-Free, Inc.
```
Comment (mbrandonw) [inline LICENSE:3]: lol!

**Review by mbrandonw — APPROVED** [review]
bummer about that linux bug :/

File: LICENSE (hunk 1 above)
Comment (stephencelis) [reply LICENSE:3]: lol! i copied from an updated repo.

**Comment by simonpierreroy** [comment]
Just to let know that the bug https://bugs.swift.org/browse/SR-10168 seems to be gone in Swift 5.1.1 on Linux 🎉

**Pull request merged**